lib/pure/math.nim

```diff
@@ -158,6 +158,23 @@ proc classify*(x: float): FloatClass =
     return fcSubnormal
   return fcNormal
 
+proc approxEqual*[T: SomeFloat](x, y: T; ulp: Natural = 4): bool {.noSideEffect.} =
```

Comment:
`almostEqual` as recommended in https://github.com/nim-lang/RFCs/issues/275#issuecomment-714290694 is same character length and not using abbreviations